pep-0008.txt

```diff
@@ -241,6 +241,17 @@ Another such case is with ``assert`` statements.
 
 Make sure to indent the continued line appropriately.
 
+
+Error Message String Literals
+-----------------------------
+
+Strive to keep error message string literals on one line,
+in order to make them easily greppable.
+If your error message exceeds the maximum line length,
+try to mitigate it by reducing indent, or just bend that rule.
```

Comment:
Maybe we don't need the "or just bend that rule." and the invocation of practicality over purity?  All of that should be well understood if you read PEP 8.

Reply:
I suppose the benefit of making it explicit is that it establishes that preserving greppability should generally take priority over following the line length restriction, which is otherwise left ambigious by the current wording and thus would seem to neuter much of the intent of the addition here, perhaps.

Reply:
I prefer the explicity of "or just bend that rule", but I will remove it if there is a strong opposition to it.